src/metrics.cpp

```diff
@@ -432,6 +432,10 @@ void ThreadShowMetricsScreen()
         std::cout << _("Thank you for running a Zcash node!") << std::endl;
         std::cout << _("You're helping to strengthen the network and contributing to a social good :)") << std::endl;
 
+        // Notify user if on testnet
+        if(CBaseChainParams::TESTNET){
+            std::cout << _("You're currently running on the Zcash testnet") << std::endl;
+        }
```

Comment:
Unrelated to this PR. Could you factor this out separately?

Reply:
Oopss.. removed.